bitcoin/src/network.rs

```diff
@@ -265,7 +331,9 @@ impl fmt::Display for Network {
 
         let s = match *self {
             Bitcoin => "bitcoin",
-            Testnet => "testnet",
+            // For user-side compatibility, testnet3 is retained as testnet
+            Testnet(TestnetVersion::V3) => "testnet",
+            Testnet(TestnetVersion::V4) => "testnet4",
             Signet => "signet",
             Regtest => "regtest",
         };
```

Comment:
This can be made a separate (private?) method and used here (Display) and in the Serialize impl, and the same for reusing FromStr logic in Deserialize.

Reply:
Dear friend, I have looked at the code, although the semantics of match is similar, serialization also relies on the data structure of serde, and it seems that it cannot be abstracted into a function